tests/api_tests/util.py

```diff
@@ -5,20 +5,36 @@
 import urlparse
 
 
+def _deep_sort(obj):
+    if isinstance(obj, dict):
+        return {
+            key: _deep_sort(value)
+            for key, value in obj.iteritems()
+        }
+    elif isinstance(obj, list):
+        return sorted([
+            _deep_sort(elem) for elem in obj
```

Comment:
It's more optimal to use a generator expression when passing to `sorted()` since it consume iterates anyway.